uportal-war/src/main/webapp/media/skins/respondr/common/less/header.less

```diff
@@ -24,6 +24,12 @@
     a {
         color: @header-link-color;
     }
+    #region-pre-header {
+       width:100%;
+       position: relative;
+       min-height:45px;
```

Comment:
I've made this choice (min-height:45px) because it will be also the sticky nav bar always on top when scrolling down. so I didn't want this bar to take many space on top on mobile view. But maybe if you want, you can add 5px.